package.json

```diff
@@ -5,6 +5,7 @@
     "node": ">= 10.13.0"
   },
   "dependencies": {
+    "sortablejs": "1.10.2",
```

Comment:
minor nit: sort alphabetically (`npm install` normally does that)

Reply:
done

Reply:
Looking good now, thanks.